### DISCUSSION TITLE
@apply not work in sass mixin on storybook production mode but works well on development

### DISCUSSION THREAD

**Discussion by ljwrer** [tailwindlabs/tailwindcss#4262] [Help]
### What version of Tailwind CSS are you using?

@tailwindcss/postcss7-compat 2.1.0

### What build tool (or framework if it abstracts the build tool) are you using?

create-react-app default

### What version of Node.js are you using?

v12.16.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

https://github.com/ljwrer/storybook-tailwind-sass-mixin-issue

### Describe your issue

tailwind.css @apply not work in sass mixin when build-storybook, but works on development mode.
when run npm storybook, you'll see a white text with blue background color in button.stories.
when run npm build-storybook, the style of text is disappear due to tailwind.css @apply not compiled.
I think the problem is the sequence of sass-loader and postcss-loader, how should I config it?

**Comment by adamwathan** [comment]
Hey! Not sure what's going on here as I'm not familiar with Storybook. Going to convert to a help thread, hopefully someone with Storybook experience can chime in 👍🏻
